Automattic/wp-calypso
# Server Settings: Add Elasticsearch static info to Cache card

Related to https://github.com/Automattic/dotcom-forge/issues/9079

## Proposed Changes
With this PR we are just adding static Elasticsearch feature indicator.


## Testing Instructions
1. Create Atomic website 
2. Open /sites -> Select atomic website -> Click on "Server settings"
3. Assert that you see the next info inside "Performance optimization" card:<br /><img width="546" alt="Screenshot 2024-09-18 at 13 03 07" src="https://github.com/user-attachments/assets/d453366e-d4e3-49d7-b0d6-4bb44aa5164c">
